api/HTMLElement.json

```diff
@@ -2451,16 +2451,21 @@
                 ]
               }
             ],
-            "firefox_android": {
-              "version_added": "29",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.w3c_pointer_events.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox_android": [
```

Comment:
This and below fix all the `onpointerxx` variants behind `dom.w3c_pointer_events.enabled` in HTMLElement to add FF 79 for Android